azure-kusto-ingest/azure/kusto/ingest/ingest_client.py

```diff
@@ -135,7 +135,13 @@ def ingest_from_blob(self, blob_descriptor: BlobDescriptor, ingestion_properties
         queues = self._resource_manager.get_ingestion_queues()
 
         authorization_context = self._resource_manager.get_authorization_context()
-        ingestion_blob_info = IngestionBlobInfo(blob_descriptor, ingestion_properties=ingestion_properties, auth_context=authorization_context)
+        ingestion_blob_info = IngestionBlobInfo(
```

Comment:
I don't think you did a complete job - see lines 128 and 129

Reply:
i deleted them and dont have them in the local version, im not really sure what happend here